content/webapp/components/HTMLSerializers/HTMLSerializers.tsx

```diff
@@ -1,90 +1,83 @@
-// @flow
 import PrismicDOM from 'prismic-dom';
 import linkResolver from './link-resolver';
-import { Fragment, type Element } from 'react';
 import { dasherize } from '@weco/common/utils/grammar';
-const { Elements } = PrismicDOM.RichText;
+import { HTMLSerializer } from 'prismic-reactjs';
+import { Fragment, ReactElement } from 'react';
 
-export type HtmlSerializer = (
-  type: string,
-  element: Object, // There are so many types here
-  content: string,
-  children: Element<any>[],
-  i: number
-) => ?Element<any>;
```

Comment:
This type is the same as one provided by `prismic-reactjs`, except theirs has `key: string` and we have `i: number`.